.pre-commit-config.yaml

```diff
@@ -12,6 +12,7 @@ repos:
       - id: check-merge-conflict
       - id: end-of-file-fixer
       - id: check-added-large-files
+        exclude: ^cluster-scope/base/openapi/
```

Comment:
Do we still need this given that the schema json now lives in a separate repository?

Reply:
Good catch; we don't need this anymore. I'll take it out.